pendant/theme.json

```diff
@@ -298,8 +315,10 @@
 			},
 			"core/site-title": {
 				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--title-font)",
 					"fontSize": "var(--wp--preset--font-size--medium)",
 					"fontWeight": "900",
+					"textTransform": "uppercase",
```

Comment:
I would rather leave off the 'uppercase'.  If a user really wants an uppercase title they can control that with their <shift> key, however if they don't with this in place they wouldn't have an option.